test/CommandLineUtils.Tests/DefaultHelpTextGeneratorTests.cs

```diff
@@ -85,5 +85,39 @@ private string GetHelpText(CommandLineApplication app)
             _output.WriteLine(helpText);
             return helpText;
         }
+        
+        enum SomeEnum { None, Normal, Extreme }
+
+        [Fact]
+        public void ShowHelp()
+        {
+            var app = new CommandLineApplication();
+            app.HelpOption();
+            app.Option("--strOpt <E>", "int option desc", CommandOptionType.SingleValue);
+            app.Option<int>("--intOpt <E>", "int option desc", CommandOptionType.SingleValue);
+            app.Option<SomeEnum>("--enumOpt <E>", "enum option desc", CommandOptionType.SingleValue);
+            app.Argument("SomeArgument1", "arg desc");
+            app.Argument<SomeEnum>("SomeArgument2", "arg desc");
+
+            StringWriter sw = new StringWriter();
```

Comment:
An easier way to do this is to reuse the `GetHelpText` method above.

Reply:
Will look into it